extensions/amp-consent/0.1/consent-ui.js

```diff
@@ -438,6 +440,7 @@ export class ConsentUI {
     toggle(dev().assertElement(this.ui_), false);
 
     const iframePromise = this.getClientInfoPromise_().then(clientInfo => {
+      clientInfo['promptTrigger'] = isActionPromptTrigger ? 'action' : 'load';
```

Comment:
Nit: Can we move the `promptTrigger'` initiation to within the `getClientInfoPromise_` function? I think it makes more sense to complete initiation within one function. Thanks.